packages/lodestar/src/chain/chain.ts

```diff
@@ -165,10 +165,17 @@ export class BeaconChain extends EventEmitter implements IBeaconChain {
     const preFinalizedEpoch = state.finalizedEpoch;
     const preJustifiedEpoch = state.currentJustifiedEpoch;
     // Run the state transition
-    const newState = stateTransition(this.config, state, block, true);
+    let newState: BeaconState;
+    try {
+      newState = stateTransition(this.config, state, block, true);
+    }catch (e) {
+      // store block root in db and terminate
```

Comment:
add logger warn or error that you caught bad block with some info like why did it fail